lib/src/page_list_viewport_gestures.dart

```diff
@@ -308,15 +341,28 @@ class DeprecatedPanAndScaleVelocityTracker {
 
   int _previousPointerCount = 0;
   int? _previousGestureEndTimeInMillis;
-  int? _previousGesturePointerCount;
 
   int? _currentGestureStartTimeInMillis;
   PanAndScaleGestureAction? _currentGestureStartAction;
   bool _isPossibleGestureContinuation = false;
 
+  // Variables needed to keep track of repeated input scroll acceleration
+  // Consider the swipe for for repeated input acceleration
+  bool _isPossibleAccelSwipe = false;
+  bool _previosLaunchedWithMomentum = true;
+  int _numberOfRepeatedAcceleratedSwipes = 0;
+  final int _maxTimeIntervalBtwRepeatedSwipes = 1000;
+  Offset _prevLaunchVelocity = Offset.zero;
+
   Offset get velocity => _launchVelocity;
+  // Initial scalar multiplying velocity in the momentum simulation
```

Comment:
This comment just confuses me. If a property needs an explanation, please make sure the explanation clarifies the meaning and the need. Also, when commenting public properties, please use Dart Docs, not regular code comments.